### PR TITLE
1.0.1 Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
#### Description
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots
- Added release notes
- Edited `.gitignore` to remove `.idea/` from the last line (`.idea/.DS_Store`)

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+| Title                          | Field containing the title of the card. The card displays the value set as the title for that record.                                                                                                                                                                                                                                                                                                                                   |
+| Subtitle                       | Field containing a short description of the card. The card displays the value set as the description of that record.                                                                                                                                                                                                                                                                                                                                                                                 |
+| Card Left Border               | Field containing the left border color code of the card. |
```
Comment (cs-dev-chaudhary) [inline docs/setup.md]: Please change it "Left border" only.

File: docs/usage.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Usage
 
-The Record Card widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the Record Card view on the Production and Development tab of the Continuous Delivery page.
+The **Record Card** widget helps visualize records based on a specified data source. An example of using the record widget would be displaying a list of tasks in the **Record Card** view on the Production and Development tab of the Continuous Delivery page.
```
Comment (cs-dev-chaudhary) [inline docs/usage.md:6]: Can we add this as an example you can use it with continuous delivery.

It looks like this widget is only meant for continuous delivery. Can we discuss this ?

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Changed

File: docs/usage.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:6]: Done.

**Review by cs-dev-chaudhary — APPROVED** [review]

**Review by cs-suyog-jinturkar — APPROVED** [review]
Dev approved it

**Pull request merged**